internal/pay/client.go

```diff
@@ -8,10 +8,13 @@ import (
 	"io"
 	"log/slog"
 	"net/http"
+	"regexp"
 
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/event"
 )
 
+var paymentsURLRe = regexp.MustCompile(`https://[a-z]+\.payments\.service\.gov\.uk/.+`)
```

Comment:
turns out it was enabled, we just were ignoring the redirects which would go to `card.payments.service.gov.uk`. I've made it more flexible to handle possible future changes